lighthouse-core/aggregator/aggregate.js

```diff
@@ -77,6 +77,12 @@ class Aggregate {
       throw new Error(msg);
     }
 
+    // Audit resulted in an error, so doesn't contribute to score.
```

Comment:
For scoring: we're doing kind of a dumb thing on master currently, adding `-1/100` as the score for audits currently returning an error result of -1.

Originally I was going to return `NaN` here for error audit scores, but that seemed unfair to do until all audits had been converted to this system (e.g. an error in user-timing would become a NaN, but an error in first-meaningful-paint would simply be `-.001`), so I manually set those to contributing 0 to match current master behavior as much as possible.

We can reconsider this (in tandem with #1512) when all audits have been converted.